src/components/Profile/components/SchedulePanel/index.tsx

```diff
@@ -48,11 +49,27 @@ const GordonSchedulePanel = ({ profile, myProf }: Props) => {
     });
   }, [profile.AD_Username]);
 
+  useEffect(() => {
+    const getAccordionState = localStorage.getItem('accordionState');
+    if (getAccordionState !== null) {
+      setAccordionState(getAccordionState === 'true');
+    }
+  }, []);
+
+  const toggleAccordionState = () => {
+    setAccordionState((a) => !a);
+    localStorage.setItem('accordionState', String(!accordionState));
```

Comment:
I'm not sure how this should be set assuming that the above change works. Perhaps it should just not be inverted?
```suggestion
    localStorage.setItem('accordionState', String(accordionState));
```

Reply:
This will exacerbate the bug. See my review for more background on why the bug exists and the best way to fix it.